docs/connector-development/connector-builder-ui/error-handling.md

```diff
@@ -0,0 +1,84 @@
+
+Note: When no backoff strategy is defined, the connection defaults to using an exponential backoff to retry requests.
+
+#### Example
+
+The [Delighted API](https://app.delighted.com/docs/api#rate-limits) is an API that recommends using an exponential backoff. In this case, the API documentation recommends retrying requests after 2 seconds, then 4 seconds, and so on. 
+
+Although a lot of API documentation does not call out using an exponential backoff, some APIs like the [Posthog API](https://posthog.com/docs/api) mention rate limits that are advantageous to use an exponential backoff. In this case, the rate limit of 240 requests/min should work for most syncs. However, if there is a spike in traffic, then the exponential backoff allows the connection to avoid sending more requests than the endpoint can support.
+
+### Wait time from header
+
+The "Wait time from header" backoff strategy allows the connection to wait before retrying a request based on the value specified in the API response.
+
+#### Example
+
+The [Chargebee API](https://apidocs.chargebee.com/docs/api/error-handling) documentation recommends using the `Retry-After` in the response headers to determine when to retry the request.
+
+When running a sync, if the API responds with a 429 error response, the connection will interpret the response and check the `Retry-After` header to see when to attempt the request again.
+
+### Wait until time from header
+
+The "Wait until time from header" backoff strategy allows the connection to wait until a specific time before retrying a request according to the API response.
+
+#### Example
+
+The [Recurly API](https://recurly.com/developers/api/v2021-02-25/index.html#section/Getting-Started/Limits) is an API that defines a header `X-RateLimit-Reset` which specifies when the request rate limit will be reset. 
+
+During a sync, if the connection receives a 429 error response, it will read the `X-RateLimit-Reset` header and pause making further requests to the Recurly API until after the time specified in the header.
+
+## Response filter
+
+A response filter should be used when a connection needs to interpret an API response to decide how the sync should proceed. Common use cases for this feature include ignoring error codes to continue fetching data, retrying requests for specific error codes, and stopping a sync based on the response received from the API.
+
+### Response conditions
+
+The following conditions can be specified on the "Response filter" and are used to determine if attributes of the response match the filter:
+- "If error message matches": The filter will match the provided text against the text body of the API response.
```

Comment:
Could we expand these three cases and have examples for them? Reading this as a novice user the following things are not clear to me:
* How is the error message determined?
* What does "matches" mean? Full match or include?
* How do I fullfill a predicate? How would that look for some example? I think especially for this one we need multiple examples because it's pretty complex
* If I specify multiple of these, is it and AND or an OR match?